docs/developer-guide/etendo-classic/how-to-guides/how-to-create-jest-testcases.md

```diff
@@ -0,0 +1,83 @@
+---
+title: How to Create Jest testcases
+---
+  
+##  Overview
+
+Jest  is a testing framework created by Facebook and currently used as the
```

Comment:
In OB wiki there are omitted links here
https://wiki.openbravo.com/wiki/How_to_create_Jest_testcases